libs/ui/src/components/atoms/typography/typography.tsx

```diff
@@ -0,0 +1,111 @@
+  font-size: ${font.size.xs};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+// ParSm Not actually implemented in design
+export const ParSm = styled.p`
+  font-size: ${font.size.sm};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParMd = styled.p`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParLg = styled.p`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const ParXl = styled.p`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+////////////////////HEADER TEXT////////////////////
+export const H6 = styled.h6`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.black};
+  font-family: ${font.family.body};
+`;
+export const H5 = styled.h5`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.bold};
+  font-family: ${font.family.body};
+`;
+export const H4 = styled.h4`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.bold};
+  font-family: ${font.family.body};
+`;
+export const H3 = styled.h3`
+  font-size: ${font.size.xxl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.body};
+`;
+export const H2 = styled.h2`
+  font-size: ${font.size.xxxl};
+  font-weight: ${font.weight.light};
+  font-family: ${font.family.body};
+`;
+export const H1 = styled.h1`
+  font-size: ${font.size.xxxxl};
+  font-weight: ${font.weight.light};
+  font-family: ${font.family.body};
+`;
+////////////////////DATA TEXT////////////////////
+export const DataXs = styled.p`
+  font-size: ${font.size.xs};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataSm = styled.p`
+  font-size: ${font.size.sm};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataMd = styled.p`
+  font-size: ${font.size.md};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataLg = styled.p`
+  font-size: ${font.size.lg};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+export const DataXl = styled.p`
+  font-size: ${font.size.xl};
+  font-weight: ${font.weight.reg};
+  font-family: ${font.family.data};
+  letter-spacing: 1px;
+`;
+///////////////////////FONT-MODS/////////////////////
+export const Light = styled.span`
+  font-weight: ${font.weight.light};
+`;
+export const Bold = styled.span`
+  font-weight: ${font.weight.bold};
+`;
+export const ExtraBold = styled.span`
+  font-weight: ${font.weight.black};
+`;
+export const Italic = styled.span`
+  font-style: italic;
+`;
+export const Underline = styled.span`
+  text-decoration: underline;
+`;
+export const StrikeThrough = styled.span`
+  text-decoration: line-through;
+`;
```

Comment:
I am wondering how things like Underline, Bold etc... are going to be used by consumers of the library. Is the expectation that they inherit those components in a new styled component and add other styles?

Reply:
Not really, they're spans that you can use to nest inside of the text components, There's examples of how this could be used in the storybook. They can be used to modify individual words or characters instead of entire blocks of text. It's hard to target individual words with css and many linters don't like it when we nest <p>'s inside of <p>'s. 

In cases where all text within a given text element needs to change its font weight, we could: 

1) wrap the inner text with the modifier

or 

2) Ask why this text element isn't in the design system. Add it to the system, name and make the new element.

Reply:
That makes sense, I was thinking more for like cases that are more complicated like bold underlined text. Would that look like:

```
<Underline>
  <Bold>
     Text
  </Bold>
</Underline>
```

Reply:
I'd imagine that would work given the structure, but I'm curious as well -- having these as `spans` makes a lot of sense so that we can have the flexibility to apply that to individual words.

Reply:
@alexkeating It works like that. If we are seeing a lot of bold, underlined words of (non-link) text, then we can make a new component.